### PR TITLE
enhance Timezone Component with Box Shadow

### LINKED ISSUE #8159: [CAL-1437] booker: missing shadow for timezone dropdown
this:

image

should be:

CleanShot 2023-04-09 at 12 48 30@2x

<sub>[CAL-1437](https://linear.app/calcom/issue/CAL-1437/booker-missing-shadow-for-timezone-dropdown)</sub>

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## What does this PR do?

i have added boxshadow to timezone options container

Fixes #8159 

**Environment**:  Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Verify that the TimezoneSelect component renders without any errors.
2. Open the dropdown menu of the TimezoneSelect component.
3 .Check that the container holding the options has the desired box shadow applied.
4. Select different options and ensure that the box shadow remains consistent.

before
beforecalcom

after
calcom

### CODE REVIEW

File: packages/ui/components/form/timezone-select/TimezoneSelect.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export function TimezoneSelect({
         menu: (state) =>
           classNames(
             "rounded-md bg-default text-sm leading-4 text-default mt-1 border border-subtle",
+            state.selectProps.menuIsOpen && "boxshadow-outline", // Add box-shadow when menu is open
```
Comment (Udit-takkar) [inline packages/ui/components/form/timezone-select/TimezoneSelect.tsx:90]: just use this property shadow-dropdown here

**Review by Udit-takkar — APPROVED** [review]
LGTM. made some changes

**Comment by sumitshinde-84** [comment]
hii @sean-brydon please review this Pull request

**Review by PeerRich — APPROVED** [review]

**Comment by sumitshinde-84** [comment]
Thank you for merging my Pull request @PeerRich .

**Pull request merged**